### PR TITLE
Add shortcut to manage summaryShown attributes in export dialogs

### PR DESCRIPTION
**Pull request by alexabird**
https://github.com/codice/ddf-ui/assets/14789712/73d2aebf-7314-4538-8d18-a63104cefb0f

### CODE REVIEW

**Comment by maiiiiiiii** [comment]
hero successful!

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/presentation.tsx [hunk 1]
```diff
@@ -70,6 +77,54 @@ const ResultsExport = (props: Props) => {
           renderInput={(params) => <TextField {...params} variant="outlined" />}
         />
       </div>
+      {['CSV', 'RTF', 'XLSX'].includes(selectedFormat) ? (
```
Comment (pvargas) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/results-export/presentation.tsx:74]: This `Manage Attributes` button looks the same as the one in [.../table-export/presentation.tsx](https://github.com/codice/ddf-ui/pull/818/files#diff-9e6f57d0c85f4b0dd6598845c56484e24289a85b568f19e2884da7bbf983be0fR125). Maybe we could have it be its own separate component?

**Review by pvargas — APPROVED** [review]

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/summary-manage-attributes/summary-manage-attributes.tsx
Comment (jrnorth) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/summary-manage-attributes/summary-manage-attributes.tsx:1]: Missing the license header.

File: ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/summary-manage-attributes/summary-manage-attributes.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                startingLeft={user
+                  .get('user')
+                  .get('preferences')
+                  .get('inspector-summaryShown')}
+                startingRight={TypedUserInstance.getResultsAttributesPossibleTable()}
+                startingHideEmpty={user
+                  .get('user')
+                  .get('preferences')
+                  .get('inspector-hideEmpty')}
+                onSave={async (active: any, newHideEmpty: any) => {
```
Comment (jrnorth) [inline ui-frontend/packages/catalog-ui-search/src/main/webapp/react-component/summary-manage-attributes/summary-manage-attributes.tsx]: ✏️ Looks like the `async` is unnecessary.

**Review by jrnorth — APPROVED** [review]

**Pull request merged**